### PR TITLE
Reproducible workflows tasks

### PR DESCRIPTION
**Pull request by takluyver**
@fangohr I've got some words in, but it's painfully slow going, so I'd appreciate some input on how well this matches what you were thinking about, and which direction to extend it.

### CODE REVIEW

**Comment by fangohr** [comment]
Hi Thomas, this is exactly what I was hoping for. Very well done. Please keep going :). 

We can and should extend this a number of ways: (i) to provide more basic information for the reader who doesn't know the tools and use cases we refer to yet. For example, what does repo2docker do? What are the use cases of having this reproducible data analysis environment? [For the latter I can think of: notebook is published with publication to re-compute all data and figures reported in the paper; with the new technology this will remain executable, and show (a) reproducibility and (b) is immediately [within a minute] re-usable if somebody wants to extend the study].

(ii) Discuss why we propose what we propose, or at least show that we have thought about this. For example, we can explain the 'keep binary docker image' versus 'keep Dockerfile' issue: Ideally, just keeping the docker file and re-creating the container from scratch when required would be best, but we can't rely on this to work for ever in the future. etc. It is okay to say that we will evaluate options as part of the tasks. We can also identify risks where we can think of them already.

(iii) Extend in the proposal style: what deliverables should we have, how much time do we allocate to each task, who else should be involved. 

For all of these, and in particular (iii) we can discuss of course.

Best wishes,

Hans

**Comment by takluyver** [comment]
I've had a go at (ii) - trying to avoid language tying us specifically to Docker, because I'm starting to become aware of other tools around containers which may be useful for this.

**Comment by takluyver** [comment]
Had a go at (i) as well.

I don't really know where to start on deliverables or time taken. I don't think I know enough about the container space yet to estimate time taken: we might find things that do what we want and just have to put them in place, or we might spend years trying to bend the available technologies to do what we want.

File: WorkPackages/jupyter-ecosystem.tex [hunk 1]
```diff
@@ -44,7 +45,12 @@
 Jupyter ecosystem projects with an emphasis on open science.
 
 repo2docker is a project for creating
-reproducible environments in which Jupyter notebooks can be run. BinderHub is software for operating a web service using repo2docker, which enables sharing of interactive and reproducible Jupyter (and Rstudio) environments on the web with a single link. We will develop repo2docker and BinderHub further to meet the needs of the open science community.
+reproducible environments in which Jupyter notebooks can be run.
```
Comment (betatim) [inline WorkPackages/jupyter-ecosystem.tex]:
```suggestion
reproducible environments in which Jupyter notebooks (and other user interfaces) can be run.
```

is this meant to be a high-level overview (short and sweet with omissions) or is being pedantic (like the above suggestion) useful here?

Comment (fangohr) [reply WorkPackages/jupyter-ecosystem.tex]: Pedantic is good.

**Pull request merged**